src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Tables/ResourceType.sql

```diff
@@ -1,6 +1,7 @@
 ﻿CREATE TABLE dbo.ResourceType
 (
     ResourceTypeId          smallint IDENTITY(1,1)          NOT NULL,
+    CONSTRAINT PK_ResourceType PRIMARY KEY NONCLUSTERED (ResourceTypeId),
     Name                    nvarchar(50)                    COLLATE Latin1_General_100_CS_AS  NOT NULL
```

Comment:
This means two copies of the table.  Would it be better to just make the PK on Name?

Reply:
That makes sense! I also talked to Katherine on this and she too agreed to have primary key on the unique column rather than identity column since we don't use identity column in any of our query. She had a good suggestion to add UNIQUE constraint for the identity column though so incorporate that change as well.